src/components/Fields/TextField.js

```diff
@@ -30,6 +30,7 @@ const propTypes = {
     max: PropTypes.number,
     min: PropTypes.number,
     name: PropTypes.string,
+    autoComplete: PropTypes.oneOf(['on', 'off']),
```

Comment:
All props in ui-react should be ordered alphabetically. Please amend here, in the defaultProps definition and in the JSX.